apps/api/src/server.ts

```diff
@@ -130,7 +130,8 @@ cron.schedule("*/5 * * * *", () => {
 
 // Run onStartUp function before starting the server
 (async () => {
-    await onStartUp();
+    // Run onStartUp function only in production
+    if (process.env.NODE_ENV === "production") await onStartUp();
```

Comment:
Could you provide the needed changes in env files in the description of this PR?

Reply:
Well the idea is, that the NODE_ENV is only set to "production" in the remote production server, and therefore the onStartUp function would only be run there. So there wouldn't be a need to add NODE_ENV to the local env files as it's an optional variable. In my head, I view the remote as the "real" production environment and the local dev:prod is something else, like a mix of development and production so the NODE_ENV shouldn't be set to "production" for the dev:prod environment. But I don't know, is this too confusing?

Reply:
Yeah, no changes needed